projects/igniteui-angular/src/lib/directives/for-of/base.helper.component.ts

```diff
@@ -104,6 +109,31 @@ export class VirtualHelperBaseDirective implements OnDestroy, AfterViewInit {
         return this.document.body.contains(this.nativeElement);
     }
 
+    private toggleClass(element: HTMLElement, className: string, add: boolean): void {
+        if (!element) return;
+        add ? this.renderer.addClass(element, className) : this.renderer.removeClass(element, className);
```

Comment:
Nitpick, but I'd use a regular control flow "if / else" statement here instead of a ternary operator.